index.html

```diff
@@ -0,0 +1,45 @@
+<<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <section id="main" class="main">
+      <h1>Dr. Norman Borlaug</h1>
```

Comment:
this should have the header tag, for semantic reasons

Reply:
- [ ] Kindly add a meaningful code and review that. Here  mentioned header tag, but in the header tag?; this project aims to learn how to do a review and learn HTML and CSS codes. In giving a code review, please ***be polite*** and clear about the inline comment you give feedback on, and explain why it required changing aspects.  Please use polite words like kindly, please, .... etc.